openfisca_us/tests/policy/baseline/income/spm_unit/spm_unit_market_income.yaml

```diff
@@ -0,0 +1,13 @@
+- name: SPM unit market income is summed person-level market income.
```

Comment:
I created this because I thought we needed a separate variable to present in the PolicyEngine accounting table. Is that right, or will it sum person-level variables up?

Reply:
It'll sum person-level variables up, yes